lucene/spatial-extras/src/test/org/apache/lucene/spatial/spatial4j/RandomizedShapeTestCase.java

```diff
@@ -183,106 +179,4 @@ private void _assertIntersect(String msg, SpatialRelation expected, Shape a, Sha
     }
   }
 
-  protected void assertEqualsRatio(String msg, double expected, double actual) {
```

Comment:
There appear to be more unused methods in this class, why did we keep them but not these?